src/components/RegionMap/RegionMap.tsx

```diff
@@ -18,18 +17,27 @@ const RegionMap: React.FC<{
   const [tooltipContent, setTooltipContent] = useState('');
 
   const countyFipsList = useMemo(() => getCountyFipsList(region), [region]);
+
+  const { result: allCountiesTopoJson } = useCountyGeographies();
+  const { result: statesTopoJson } = useStateGeographies();
+
   const countiesTopoJson = useMemo(
-    () => buildCountyGeometries(countyFipsList),
-    [countyFipsList],
-  );
-  const projectionConfig = useMemo(
-    () => getProjectionConfig(countiesTopoJson, width, height),
-    [countiesTopoJson, width, height],
+    () =>
+      allCountiesTopoJson
+        ? buildCountyGeometries(allCountiesTopoJson, countyFipsList)
+        : null,
+    [countyFipsList, allCountiesTopoJson],
   );
 
-  const statesTopoJson = useMemo(() => getStateGeometries(countyFipsList), [
-    countyFipsList,
-  ]);
+  const projectionConfig = useMemo(() => {
```

Comment:
This isn't fetching anything is it? I would place this after the if statement
```
if (!statesTopoJson || !countiesTopoJson) { return null; }
const projectionConfig = useMemo(() => {
  return getProjectionConfig(...);
}, [...])
```

Reply:
`useMemo` cannot be called conditionally. I'm memoizing that call because there is a bunch of math going on in calculating the centroid and scale for a given set of county TopoJSON objects, so we probably don't want to run that too often.

Reply:
cool! Could you add that as a comment about why useMemo is used there?